### PR TITLE
Add SemanticService.tooltip method for Android tooltips

### LINKED ISSUE #16943: Tooltips should emit TYPE_WINDOW_STATE_CHANGED on Android

### PR DESCRIPTION
**Pull request by jonahwilliams**
Fixes https://github.com/flutter/flutter/issues/16943. Requires https://github.com/flutter/engine/pull/5103

### CODE REVIEW

**Comment by jonahwilliams** [comment]
Bad sync

**Comment by jonahwilliams** [comment]
Fixed

**Review by goderbauer — COMMENTED** [review]
Will the appearance of the tooltip now trigger a sound on iOS? Is that intended? 

Also on Android, when the Tooltip disappears there will now be an announcement of the route below it. Is that ok? Is that what native does?

**Comment by jonahwilliams** [comment]
Fixed with discussed changes

File: packages/flutter/lib/src/material/tooltip.dart [hunk 1]
```diff
@@ -147,6 +147,13 @@ class _TooltipState extends State<Tooltip> with SingleTickerProviderStateMixin {
     _entry = new OverlayEntry(builder: (BuildContext context) => overlay);
     Overlay.of(context, debugRequiredFor: widget).insert(_entry);
     GestureBinding.instance.pointerRouter.addGlobalRoute(_handlePointerEvent);
+    switch (defaultTargetPlatform) {
```
Comment (goderbauer) [inline packages/flutter/lib/src/material/tooltip.dart]: I'd probably always send this event and leave it up to the platform side to either drop it on the floor or announce it.

**Review by goderbauer — APPROVED** [review]
LGTM

**Pull request merged**